bundles/CoreBundle/Command/Document/MigrateElementsCommand.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Command\Document;
+
+use Doctrine\DBAL\Query\QueryBuilder;
+use Pimcore\Console\AbstractCommand;
+use Pimcore\Db;
+use Pimcore\Document\Tag\NamingStrategy\NamingStrategyInterface;
+use Pimcore\Document\Tag\NamingStrategy\NestedNamingStrategy;
+use Pimcore\Model\Document\Targeting\TargetingDocumentInterface;
+use Pimcore\Model\Element\Recyclebin\Item;
+use Pimcore\Model\Version;
+use Pimcore\Tool\Serialize;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+
+class MigrateElementsCommand extends AbstractCommand
+{
+    /**
+     * @var bool
+     */
+    private $runCommand = true;
+
+    protected function configure()
+    {
+        $this
+            ->setName('pimcore:documents:migrate-elements')
+            ->setDescription('Migrates document elements to editables. See issue https://github.com/pimcore/pimcore/issues/7384 first');
```

Comment:
how about we add an option to restrict migration to a particular set? like this https://github.com/pimcore/pimcore/blob/6.9/bundles/CoreBundle/Command/Migration/ControllerReferenceMigrationCommand.php#L61